hw/ip/adc_ctrl/doc/_index.md

```diff
@@ -1,34 +1,215 @@
 ---
-title: "ADC_CTRL HWIP Technical Specification"
+title: "Analog to Digital Converter Control Interface"
 ---
 
 # Overview
 
-
+This document specifies the ADC controller IP functionality.
+This IP block implements control and filter logic for an analog block that implements a dual ADC.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification" >}})
+See that document for integration overview within the broader top level system
 
 ## Features
 
+The IP block implements the following features:
 
+- Register interface to dual ADC analog block
+- Support for 2 ADC channels
+- Support for 8 filters on the values from the channels
+- Support ADCs with 10-bit output (two reserved bits in CSR)
+- Support for debounce timers on the filter output
+- Uses slow clock to enable use while the part is sleeping
```

Comment:
Maybe rephrase as `Runs on a slow always-on clock to enable usage while the device is sleeping`

Reply:
Sure.